Automattic/jetpack
# CRM: Adding module exports for vars and all functions

## Proposed changes:

* This PR adds exports for various variables throughout CRM that are referenced via 'window.variablename'. This led to the following reported error for one particular variable: https://github.com/Automattic/zero-bs-crm/issues/2878
* It also includes all remaining functions (not just those called from files outside of the file thy are declared in), as well as any other globally declared variables.


### Other information:


## Jetpack product discussion

pbhBOz-2YT-p2#comment-3740

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* To test on Jurassic Ninja, create a new Jurassic Ninja site with the [Beta plugin](https://github.com/Automattic/jetpack-beta) installed and active, and under Jetpack CRM on the Beta testing dashboard add this feature branch.
* The issue reported in https://github.com/Automattic/zero-bs-crm/issues/2878 should no longer exist.
